test/hyprland.conf

```diff
@@ -1,20 +1,12 @@
-$var1=var1
-$var2=var2
-$var3=var3
-$var4=var4
-
-$MODKEY=SUPER
-
-bind=MOD, L, exec, nospace
-bind=, L, exec, space
-
-bindm=SUPER,mouse:420,movewindow
-bindm=, mouse:69, resizewindow
-
-bind=$MODKEY,L,exec,$var1
-bind=$MODKEY, R, exec, $var2
-bind=,R,exec,$var3
-bind=, R, exec, $var3
-
-bindlm=, mouse:69, resizewindow2
-bindlrfuckme=, mouse:69, resizewindow2
```

Comment:
is there a reason you removed all of those? The standard arguments (`--verbose` and `--markdown`) both depend on those to be in the test config.